src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -222,6 +230,51 @@ def get_line_angles_with_yaxis(self, pose_dix):
         
         return (l_angle, r_angle)
 
+    def estimate_spinalVector_score(self, pose):
+        pose_dix = {}
+
+        # Calculate leftVectorScore & rightVectorScore
+        leftVectorScore = min(pose.keypoints['left shoulder'].score, pose.keypoints['left hip'].score)
+        rightVectorScore = min(pose.keypoints['right shoulder'].score, pose.keypoints['right hip'].score) 
+
+        if leftVectorScore > self.confidence_threshold:
+            pose_dix['left shoulder'] = pose.keypoints['left shoulder'].yx
+            pose_dix['left hip'] = pose.keypoints['left hip'].yx
+
+        if rightVectorScore > self.confidence_threshold:
+            pose_dix['right shoulder'] = pose.keypoints['right shoulder'].yx
+            pose_dix['right hip'] = pose.keypoints['right hip'].yx
+        
+        def find_spinalLine():
+            left_spinal_x1 = (pose_dix['left shoulder'][0] + pose_dix['right shoulder'][0]) / 2
+            left_spinal_y1 = (pose_dix['left shoulder'][1] + pose_dix['right shoulder'][1]) / 2
+            
+            right_spinal_x1 = (pose_dix['left hip'][0] + pose_dix['right hip'][0]) / 2
+            right_spinal_y1 = (pose_dix['left hip'][1] + pose_dix['right hip'][1]) / 2
+            
+            return (left_spinal_x1, left_spinal_y1), (right_spinal_x1, right_spinal_y1)
+
+        is_leftVector = leftVectorScore >= self.confidence_threshold
```

Comment:
@bhavikapanara maybe we can bump `is_leftVector` a little higher in this function to remove the redundancy of `rightVectorScore > self.confidence_threshold`. Same for the left side.

Reply:
Done